elyra/util/operation_parser.py

```diff
@@ -111,53 +99,89 @@ def __init__(self, operation_filepath):
     def type(self):
         raise NotImplementedError()
 
+    @property
+    def model(self):
+        return self.build_model()
```

Comment:
I would suggest something more like the following so we don't build the results again if the `model` property is accessed multiple times.
```python
self._model = None

def model(self):
    if not self._model:
        self._model = self.build_model()
    return self._model
```